app/controllers/users_controller.rb

```diff
@@ -1,7 +1,14 @@
 class UsersController < ApplicationController
+  # The index action is responsible for displaying a list of users. It fetches all user records from the database
+  # assigns them to an instance variable @users.
+  # This instance variable can be accessed within the corresponding view file to display the list of users.
   def index
     @users = User.all
   end
 
-  def show; end
+  # the show action retrieves a user record from the database
+  # based on the id parameter passed in the request (params[:id]).
+  def show
+    @user = User.find(params[:id])
+  end
 end
```

Comment:
- You have done great so far adding the actions and the corresponding view files, but there are still a few requirements you need to meet before moving to the next milestone. :+1: :100: 

- [ ] Looking at the video below, You can see that when I start the app and open the `#root` URL, I get an error, this is because you are pointing the root to the post index, but the post index needs a user a user id from the URL param, so I'll suggest you make the users index the root URL. 
- [ ] Also when I navigate to `/users` I got another error different from the first one. I'll suggest you actually fix these before requesting another review. :+1: :+1:  

- - Video reference:
https://drive.google.com/file/d/1c0ZFbG03pv6mo0q23c_zTrYFBXYNFVr1/view?usp=sharing